pkg/promlib/converter/prom.go

```diff
@@ -21,7 +21,6 @@ func logf(format string, a ...any) {
 }
 
 type Options struct {
```

Comment:
Nit: It crossed my mind that we could remove this Options object too, and make this code even simpler. But maybe we keep it, in anticipation of future options? Just an idea - I'll leave it up to you.